auth.py

```diff
@@ -14,6 +14,7 @@
 WOTT_SCREENLY_CREDENTIAL_NAME = 'screenly'
 
 
+
```

Comment:
this causes our build to fail with PEP8 error

Reply:
fixed